### PR TITLE
Ensure task inputs are consistent for checkstyle

### PR DESCRIPTION
**Pull request by jrhee17**
Motivation:

We are passing `configProperties` to the checkstyle task, which is resulting in inconsistent inputs to our gradle task when run from different locations.
By setting the `configDirectory` property, we can use the `config_loc` property pre-defined by `checkstyle`.

ref: https://ge.armeria.dev/scans?search.tags=exp3*%2Cgradle/checkstyle-cache&search.timeZoneId=Asia/Seoul#selection.buildScanA=r4jnvs4tiabmg&selection.buildScanB=zhqrycgjdns7q

Modifications:

- Specify `configDirectory` instead of `configProperties`

Result:

- Input caching is done properly from different locations for checkstyle tasks.

### CODE REVIEW

**Review by trustin — APPROVED** [review]
:+1:

File: settings/checkstyle/checkstyle.xml [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
   <module name="SuppressWarningsFilter" />
   <module name="SuppressionFilter">
-    <property name="file" value="${checkstyleConfigDir}/checkstyle-suppressions.xml"/>
+    <property name="file" value="${config_loc}/checkstyle-suppressions.xml"/>
```
Comment (minwoox) [inline settings/checkstyle/checkstyle.xml:13]: So `config_loc` is a built-in property that gets mapped to the directory the configuration file lies in.

**Review by minwoox — APPROVED** [review]
Thanks! 👍 👍 👍

File: settings/checkstyle/checkstyle.xml (hunk 1 above)
Comment (jrhee17) [reply settings/checkstyle/checkstyle.xml:13]: I believe `configDirectory` is the value injected into `config_loc`

Comment (minwoox) [reply settings/checkstyle/checkstyle.xml:13]: Oops, you already mentioned it in the commit description. Sorry about it. 😓

**Review by ikhoon — APPROVED** [review]
Looks nice!

**Pull request merged**